src/Components/SavedSearchAlert/Components/AddFiltersWaysToBuy.tsx

```diff
@@ -0,0 +1,75 @@
+import { Box, Checkbox, Flex, Spacer, Text } from "@artsy/palette"
+import { entries } from "lodash"
+import { FC } from "react"
+
+import { useSavedSearchAlertContext } from "Components/SavedSearchAlert/SavedSearchAlertContext"
+import { SearchCriteriaAttributes } from "__generated__/createSavedSearchAlertMutation.graphql"
+
+interface WayToBuy {
+  selected: boolean
+  name: string
+  key: keyof SearchCriteriaAttributes
+}
+
+export const WAYS_TO_BUY_OPTIONS = {
```

Comment:
Same, could we use https://github.com/artsy/force/blob/20165eac3335ff5e4a352d63cc4f7d394a51e210/src/Components/SavedSearchAlert/Components/AddFiltersWaysToBuy.tsx#L14?